### ISSUE TITLE
Tool-tip for lids on beakers and bottles is inverted

### ISSUE THREAD

**Issue by GGlut** [RMC-14/RMC-14#4807]
## Description
Tool-tip for lids on beakers and bottles is inverted. It says it has a lit when its open.

**Screenshots**
![Screenshot 2024-11-13 022924](https://github.com/user-attachments/assets/f9a87bd6-f3cf-4749-ae6f-8a91308ebf49)
![Screenshot 2024-11-13 022932](https://github.com/user-attachments/assets/77778278-e20d-4d04-b9d8-cffa2b756068)

**Comment by Vermidia** [comment]
yeah this is my bad, the text needs to be inverted to say it doesn't have any lid or something.
